src/solvers/refinement/refine_arrays.cpp

```diff
@@ -100,8 +100,6 @@ void bv_refinementt::arrays_overapproximated()
         false,
         string_refinement_invariantt("error in array over approximation "
           "check"));
-      // Placeholder to tell the compiler we bail
-      throw 0;
```

Comment:
I haven't bothered to do a `git blame`, but really the code above these lines should be cleaned up as well: no duplicate text strings, please. The `error()` line should just go away.